buyer/logic.js

```diff
@@ -0,0 +1,4 @@
+function search(array, value) {
+  let newArr = array.filter((item) => item.name.includes(value))
```

Comment:
If the word is entered with a space at its beginning or end, make him recognize it and delete the space.